backend/Tournament/views.py

```diff
@@ -0,0 +1,131 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from django.contrib.auth import authenticate
+from django.utils.decorators import method_decorator
+
+from .models import Tournament, \
+						TournamentPlayer
+from .serializers import TournamentSerializer, \
+	 						TournamentCreationSerializer, \
+							TournamentPlayerSerializer, \
+							TournamentInProgressSerializer
+from .managers import TournamentSetupManager, \
+						TournamentInProgressManager
+from .exceptions import TournamentSetupException, \
+							TournamentInProgressException
+from .tournamentState import TournamentState
+from User.models import User
+from shared_utilities.decorators import must_be_authenticated, \
+											must_not_be_username, \
+											check_that_valid_tournament_request
+
+import sys
+
+# Create your views here.
+class TournamentListView(APIView):
+	@method_decorator(must_be_authenticated)
+	def post(self, request):
+		body_data = request.data.copy()
```

Comment:
I use copy because you can't modify the request. so I just copy the the body and add the host user into the it before validating the serializer.

of course the frontend could just send the host user with the request, but we can get it in the backend regardless